include/libopencm3/stm32/h7/rcc.h

```diff
@@ -200,6 +439,37 @@ enum rcc_osc {
   RCC_LSI
 };
 
+enum rcc_sysclk_mux {
+  RCC_SYSCLK_PLL,
+  RCC_SYSCLK_HSE,
+  RCC_SYSCLK_HSI,
```

Comment:
how are these differrent to the defines that every other platforrm has for sysclock sources?  Why does all of this platform need a sea of enums?  (and enums that are often just equal to some defines?)